apps/meteor/server/modules/listeners/listeners.module.ts

```diff
@@ -209,6 +209,12 @@ export class ListenersModule {
 		});
 
 		service.onEvent('watch.inquiries', async ({ clientAction, inquiry, diff }): Promise<void> => {
+			// We do not need inquiries on the client when the routing method is not manual
+			// When the routing method changes, client fetches inquiries again and discards the received ones
+			if (settings.get('Livechat_Routing_Method') !== 'Manual_Selection') {
```

Comment:
The thing is like this:
On the UI, when the algo is manual selection, a fix was made so the client unsubscribes correctly to the inquiries stream (this was made in other PR about queue refactor, since this was a BE problem) so the client _shouldnt be expecting to receive the inquiries_ when they're not on manual.

The fix is more to stop sending them into the void or to places that could be subscribed, but the UI shouldn't be receiving them anymore.